tests/e2e/Services/VCS/VCSConsoleClientTest.php

```diff
@@ -0,0 +1,319 @@
+<?php
+
+namespace Tests\E2E\Services\VCS;
+
+use Tests\E2E\Scopes\Scope;
+use Tests\E2E\Client;
+use Tests\E2E\Scopes\ProjectCustom;
+use Tests\E2E\Scopes\SideConsole;
+use Utopia\App;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Helpers\Role;
+use Utopia\VCS\Adapter\Git\GitHub;
+use Utopia\Cache\Adapter\None;
+use Utopia\Cache\Cache;
+
+class VCSConsoleClientTest extends Scope
```

Comment:
VCS tests won't work locally, until you set up some VCS env variables, right? For example stuff like secret keys we cant simply have in .env.

@christyjacob4 do you think we should see if secrets for VCS tests are missing, and if so, skip it? That way it doesn't run (and doesn't fail) locally, but it still runs in CI/CD. And if it fails on CI/CD and you want to investigate, you can set variables.

Reply:
I am thinking something like this, in VCSBase: https://github.com/appwrite/appwrite/blob/073b9963d20945f4f1b6c6b35284fc3007940062/tests/e2e/General/AbuseTest.php#L20-L27

Reply:
(in PM Christy confirmed we should try to do this)